rendermodules/lens_correction/apply_lens_correction.py

```diff
@@ -83,6 +104,8 @@ def run(self):
 
             for ts in tspecs:
                 ts.tforms = [ref_lc] + ts.tforms
+                for i in range(len(mask_mm_list)):
```

Comment:
This makes an assumption about the levels available in the tilespec -- something like the following would fit the create_mipmaps file better
```python
for lvl, maskUrl in mask_mm_list.items():  # mask_mm_list is a dictionary
    ts.ip[lvl].maskUrl = pathlib.Path(maskUrl).as_uri()
```